src/pages/GivePage/index.tsx

```diff
@@ -748,69 +735,7 @@ const AllocateContents = ({
           })}
       </Panel>
 
-      {filteredMembers.length == 0 && (
-        <Box>
-          <Flex css={{ justifyContent: 'center', py: '$3xl' }}>
-            <Text
-              inline
-              css={{ textAlign: 'center' }}
-              color="secondary"
-              h3
-              semibold
-            >
-              {onlyCollaborators
-                ? "You don't have any collaborators yet."
-                : 'No members in this circle.'}
-            </Text>
-          </Flex>
-          <Flex css={{ justifyContent: 'center' }}>
-            <Box
-              css={{
-                position: 'relative',
-                width: '80%',
-                '@md': {
-                  width: '90%',
-                },
-              }}
-            >
-              <Box css={{ zIndex: 1, width: '100%' }}>
-                {/* this is an example for documentation not a real allocation row*/}
-                <GiveRow
-                  member={exampleHelpMember}
-                  updateTeammate={async () => {}}
-                  adjustGift={() => {}}
-                  gift={{
-                    tokens: 0,
-                    recipient_id: -300,
-                  }}
-                  maxedOut={false}
-                  setSelectedMember={() => {}}
-                  noGivingAllowed={true}
-                  docExample={true}
-                  selected={false}
-                />
-                <Flex css={{ justifyContent: 'center', pt: '$lg' }}>
-                  <Text css={{ width: '15em' }}>
-                    To add collaborators select a member and add them as a
-                    collaborator.
-                  </Text>
-                </Flex>
-              </Box>
-              <Box
-                css={{
-                  position: 'absolute',
-                  width: '100%',
-                  height: '100%',
-                  top: 0,
-                  left: 0,
-                  zIndex: 2,
-                }}
-              ></Box>
-            </Box>
-          </Flex>
-        </Box>
-      )}
-
+      {}
```

Comment:
Whatsup with this change? This deletes the example of how to add a contributor, which was in the design

Reply:
will replace.